Tests/UnitTests/MessageBirdUnitTests/Resources/ConversationTest.cs

```diff
@@ -23,7 +23,7 @@ public void Start()
                 .ThatExpects(
                     @"{""to"":""+31612345678"",""type"":""text"",""content"":{""text"":""Hello!""},""channelId"":""619747f69cf940a98fb443140ce9aed2""}")
                 .AndReturns(filename: "ConversationView.json")
-                .FromEndpoint("POST", "conversations/start", ConversationsResource.ConverstationsBaseUrl)
+                .FromEndpoint("POST", "conversations/start", VoiceCallResource.ConverstationsBaseUrl)
```

Comment:
This class name shouldn't be changing, this needs to stay `ConversationsResource`